src/punctuation2.js

```diff
@@ -10,11 +10,11 @@
 *          (They are jumped over when looking for next (non-controlfield subfield)
 */
 import {validateSingleField} from './ending-punctuation';
-// import createDebugLogger from 'debug';
-import {fieldToString, nvdebug} from './utils';
+import createDebugLogger from 'debug';
+import {fieldToString, nvdebug, subfieldToString} from './utils';
 import clone from 'clone';
 
-// const debug = createDebugLogger('@natlibfi/marc-record-validators-melinda/punctuation2');
+const debug = createDebugLogger('@natlibfi/marc-record-validators-melinda/punctuation2');
```

Comment:
Could this be changed to something like 
`const debug = createDebugLogger('debug/punctuation2');` so it wont be triggered by `@natlibfi/*` where flow debug is?

Reply:
Yes.

Reply:
(I've only seen createDebugLogger used in @natlibfi style, so I have followed that convention here.

Reply:
Commented out two debug messages, and this became irrelevant (for now). Also changed the out-commented createdDebugLogger() function call,